lib/scrub/scrub_runner.rb

```diff
@@ -60,17 +53,36 @@ def run_file(file)
       downloaded_file = download_to_work_dir(file)
       scrubber = Scrub::AutoScrub.new(downloaded_file)
       scrubber.run
+
+      rc = Scrub::RecordCounter.new(@organization, scrubber.item_type)
+      unless rc.acceptable_diff? || @force
+        raise MalformedFileError, [
```

Comment:
A couple questions here:

- Should `MalformedFileError` know how to generate this message itself (given a `Scrub::RecordCounter`?)
- Should `Scrub::RecordCounter` raise the exception itself (perhaps something like `rc.ensure_acceptable_diff!`)?

Reply:
* I don't think so
* I don't think so

But I could be convinced otherwise.

Reply:
`run` is definitely starting to get a bit on the lengthy side with the logic for several different things here happening. The two options I see that would help with that are as described above (at least in terms of `MalformedFileError` knowing how to generate this message) and the innards of the block starting here https://github.com/hathitrust/holdings-backend/pull/226/commits/5d5aab2edba1c0ee5efcd50cebb8e1298973a5be#diff-ca99b54ef7e5ac35271f7440dc7cf9ee2214ddd251430e6c57d7e38138feee8cL64 (as previously suggested). I'd certainly be hesitant to make more changes to `run` here without extracting some of the pieces.